samples/nrf9160/modem_shell/src/link/link_shell_pdn.c

```diff
@@ -202,17 +274,25 @@ int link_shell_pdn_connect(const struct shell *shell, const char *apn_name,
 	/* TODO: Set authentication params if requested by using pdn_ctx_auth_set() */
 
 	/* Activate a PDN connection */
-	ret = pdn_activate(cid, &esm);
+	ret = link_shell_pdn_activate(shell, cid);
 	if (ret) {
-		shell_error(shell, "pdn_activate() failed, err %d esm %d %s\n",
-			    ret, esm, esm_strerr(esm));
 		goto cleanup_and_fail;
 	}
 
+	pdn_id = pdn_id_get(cid);
+
+	/* Add to PDN list: */
+	new_pdn_info = link_shell_pdn_info_list_add(pdn_id, cid);
+	if (new_pdn_info == NULL) {
+		shell_error(
+			shell,
+			"ltelc_pdn_init_and_connect: could not add new PDN socket to list\n");
```

Comment:
Is `ltelc_pdn_init_and_connect` correct in this context? It doesn't seem to me, unless I miss something.

Reply:
typo, fixed